community/examples/htcondor-pool.yaml

```diff
@@ -0,0 +1,136 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+blueprint_name: htcondor-pool
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: htcondor-001
+  region: us-central1
+  zone: us-central1-c
+
+deployment_groups:
+- group: htcondor
+  modules:
+  - source: modules/network/vpc
+    kind: terraform
+    id: network1
+    settings:
+      network_name: htcondor-pool
+      subnetwork_name: htcondor-pool-usc1
+    outputs:
+    - network_name
+
+  - source: community/modules/scripts/htcondor-install
+    kind: terraform
+    id: htcondor_install
+
+  - source: community/modules/project/service-enablement
+    kind: terraform
+    id: htcondor_services
+    use:
+    - htcondor_install
+
+  - source: community/modules/scheduler/htcondor-configure
+    kind: terraform
+    id: htcondor_configure
+
+  - source: modules/scripts/startup-script
+    kind: terraform
+    id: htcondor_configure_central_manager
+    settings:
+      runners:
+      - type: shell
+        source: modules/startup-script/examples/install_ansible.sh
+        destination: install_ansible.sh
+      - $(htcondor_install.install_htcondor_runner)
+      - $(htcondor_configure.central_manager_runner)
+
+  - source: modules/scripts/startup-script
+    kind: terraform
+    id: htcondor_configure_access_point
+    settings:
+      runners:
+      - type: shell
+        source: modules/startup-script/examples/install_ansible.sh
+        destination: install_ansible.sh
+      - $(htcondor_install.install_htcondor_runner)
+      - $(htcondor_install.install_autoscaler_deps_runner)
+      - $(htcondor_install.install_autoscaler_runner)
+      - $(htcondor_configure.access_point_runner)
+      - $(htcondor_execute_point.configure_autoscaler_runner)
```

Comment:
nit: At one point we had talked about only being able to reference modules that had already been defined above (as a cheap way to avoid cycles). This would violate. Maybe move each startup script next to the module that is consuming them.

Reply:
Performed re-org due to increased clarity of connection.

Reply:
Addressing this nit requires a re-approval @nick-stroud